spectator-api/src/main/java/com/netflix/spectator/api/patterns/GaugePoller.java

```diff
@@ -47,6 +48,10 @@
   private static final ScheduledExecutorService DEFAULT_EXECUTOR =
       Executors.newSingleThreadScheduledExecutor(FACTORY);
 
+  static ScheduledFuture<?> schedule(long delay, Runnable task) {
```

Comment:
I noticed in one of the other PRs that there were similar methods covering both Callable and Runnable. Should that be done here as well?